Venmo/src/test/java/com/braintreepayments/api/VenmoApiUnitTest.java

```diff
@@ -168,6 +168,64 @@ public void createPaymentContext_whenGraphQLPostError_forwardsErrorToCallback()
         verify(callback).onResult(null, error);
     }
 
+    @Test
+    public void createPaymentContext_withTotalAmountAndSetsFinalAmountToTrue() throws JSONException {
```

Comment:
Does it work ok if total amount is not set and final amount is set to true? No error just doesn't show total amount at all?

Reply:
The only use case I'd be concerned about is if total amount not set, final amount true (unsure what the behavior would be for that), but as long as it is still ok then, Probably every combination doesn't need a separate unit test

Reply:
Yep, total amount is optional for both `VenmoPaymentMethodUsage.SINGLE_USE` and `VenmoPaymentMethodUsage.MULTI_USE`. I just tested both cases with only setting `setIsFinalAmount(true)` without `setTotalAmount` and there are no issues.